README.md

```diff
@@ -23,6 +23,11 @@ Find out more about configuration/usage
 
 * [`/scripts/mkcloud`](docs/mkcloud.md)
 
+# License
+
+Files in this repository are licensed under the GNU LGPL 2.1 license,
+unless stated otherwise. See [the LICENSE file](LICENSE) for details.
```

Comment:
There are headers in some scripts that are using GPL, that I guess that is covered by this `otherwise`. Are we going to change those other scripts in the repo?